src/topics/inputs.md

```diff
@@ -190,11 +190,21 @@ You can specify arrays of arrays, arrays of records, and other complex types.
 
 ## Inclusive and Exclusive Inputs
 
-Sometimes an underlying tool has several arguments that must be provided
-together (they are dependent) or several arguments that cannot be provided
-together (they are exclusive).  You can use records and type unions to group
+Sometimes an underlying tool has several arguments that must all be provided
+together (they are inclusive/dependent) or several arguments that cannot be provided
+together, only one of the arguments can be provided (they are exclusive).  You can use records and type unions to group
 parameters together to describe these two conditions.
 
+```{image} /_static/images/diagrams/inclusive_v_exclusive.png
+:name: inclusive_v_exclusive-diagram
+:width: 600px
+:align: center
+```
+
+
```

Comment:
I think the extra blank lines are not necessary. This is not really important, but we tried to remove unnecessary blank lines before, so probably better if we can avoid adding extra lines that are not necessary.